app/views/shared/_footer_lite.html.erb

```diff
@@ -7,11 +7,14 @@
   <% if show_language_dropdown %>
     <div class='tablet:display-none border-bottom border-primary-light'>
       <div class='container cntnr-wide padding-y-1 padding-x-2 desktop:padding-x-0 h5'>
-        <div class='i18n-mobile-toggle center'>
-          <%= link_to('#', class: 'block text-decoration-none blue fs-13p', 'aria-expanded': 'false') do %>
-            <%= image_tag asset_url('globe-blue.svg'), width: 12, class: 'margin-right-1', alt: '',
-              'aria-hidden': 'true' %><%= t('i18n.language') %><span class='caret inline-block ml-tiny' aria-hidden='true'>&#9662;</span>
-          <% end %>
+        <div class='flex flex-align-center flex-justify-center'>
+          <div class='i18n-mobile-toggle center'>
```

Comment:
Is `center` necessary now that we're centering with flexbox?

Asking partly because `center` is one of the BassCSS classes slated for removal (LG-3878), and removing it now would make that work easier in the future.

Reply:
Also, could the two `div`s be merged together to a single `<div class="i18n-mobile-toggle flex flex-align-center flex-justify-center">` ?